### ISSUE TITLE
#Suggestion - Variables

### ISSUE THREAD

**Issue by iijake** [pyrogram/pyrogram#226]
If you want to add a variables like **Chat_Id** you need to write **Chat_Id = message.chat.id** 
it will be better if there is class like :
_**from pyrogram import Variables**_

and not need to use **Chat_Id = message.chat.id** , you just write **Chat_Id** 
or if you want **File_ID** you do not need you use **message.video.file_id** you just write **Video_File_ID**

HERE ALL THE VARIABLES : 
- https://core.telegram.org/bots/api#available-types

**Comment by delivrance** [comment]
@iijake This is doable technically, but makes very little sense to me. I've never seen such usage anywhere else and I'm disliking it at first sight. Do you have any example of such usage pattern from any widely used python library/framework?

**Comment by iijake** [comment]
i don't see python library/framework use it , but i see it in php, Probably because it is webhook

https://github.com/davtur19/TuriBot

**Comment by delivrance** [comment]
tl;dr: Nope, this is ugly. So many things wrong. Use dot notation and save your properties if you need to re-use them.

First of all, blame PHP for not having a dot notation for object properties, `message.chat.id` is perfectly fine in Python and many other languages.

Next, your naming style is screwed up and your suggestion would introduce a mess of a code hard to maintain that would very much likely look like a bad language hack for no real benefits: you are going to save just few bytes this way (if any, since you'd need to access Variables every time, unless you want to import generic names like message_id, chat_id or, worse, import everything and pollute your namespace), while confusing people by introducing an ugly and unintuitive interface.
 
<details><summary>Ouch!</summary>
<a href="https://github.com/davtur19/TuriBot/blob/b4418cd895289c465b5ea689b8165f250608f723/bot.php">https://github.com/davtur19/TuriBot/blob/b4418cd895289c465b5ea689b8165f250608f723/bot.php</a>
<img src="https://user-images.githubusercontent.com/14043624/54160750-ae120600-4450-11e9-88da-b03139e6239b.png">
</details>

**Comment by davtur19** [comment]
I confirm what @delivrance said, I never liked that piece of code, and when I had time, I rewrote the whole code 😅

> you are going to save just few bytes this way

however, with the code I used, more memory was also used
